sources/representation/qt/mainform.cpp

```diff
@@ -202,13 +220,13 @@ void MainForm::startGameLoop(int id, QString map)
             addSystemText("FPS", "FPS: " + QString::number(fps_counter));
             addSystemText(
                 "{Perf}Represent",
-                  "Represent max: "
+                "Represent max: "
```

Comment:
Here should be extra 2 spaces.
The logic is next - when there are multiple lines in the expression then each operand should start with same position.
```
  operand1
+ operand2
+ operand3
```
So a first operand usually gets some extra spaces (here it is `"Represent max: "`).

But here actually it will be better to rewrite the old code using `QString`:
```
QString("Represent max: %1 ms").arg(max_process_time / 1e6);
```